ios-base/AppDelegate.m

```diff
@@ -1,17 +1,18 @@
 //
 //  AppDelegate.m
-//  ios-base
-//
-//  Created by Danis Ziganshin on 14.02.14.
-//  Copyright (c) 2014 FlatStack. All rights reserved.
 //
 
 #import "AppDelegate.h"
 
 @implementation AppDelegate
 
-- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+- (BOOL)application:(UIApplication *)application
+didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
```

Comment:
It's nice in functions with many arguments, but not when we have 1-2 incoming values